OrbitClientProtos/orbit_log_event.proto

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+syntax = "proto3";
+
+package orbit_client_protos;
+
+message OrbitLogEvent {
+  enum LogEventType {
+    UNKNOWN_EVENT_TYPE = 0;
+    ORBIT_INITIALIZED = 1;
+  }
+  LogEventType log_event_type = 1;
+}
```

Comment:
That's the proto which have to be kept in sync with the library, isn't it? Could you add a comment explaining that to avoid folks accidentally changing that without making the some change in the library?

Reply:
Yes, right, added a comment about it, thank you!